src/shared/components/VideoPlayer/VideoPlayer.style.tsx

```diff
@@ -1,12 +1,178 @@
 import { css } from '@emotion/react'
 import styled from '@emotion/styled'
 
-import { colors, media, sizes, transitions, typography, zIndex } from '../../theme'
+import { SvgPlayerSoundOff } from '@/shared/icons'
+
+import { colors, sizes, transitions, zIndex } from '../../theme'
+import { Text } from '../Text'
 
 type ContainerProps = {
   isInBackground?: boolean
+  isFullScreen?: boolean
+}
+type CustomControlsProps = {
+  isFullScreen?: boolean
 }
 
+const focusStyles = css`
+  :focus {
+    /* Provide a fallback style for browsers
+     that don't support :focus-visible e.g safari */
+    box-shadow: inset 0 0 0 3px ${colors.transparentPrimary[18]};
+  }
+
+  /* https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible */
+
+  :focus-visible {
+    box-shadow: inset 0 0 0 3px ${colors.transparentPrimary[18]};
+  }
+
+  :focus:not(:focus-visible) {
+    box-shadow: unset;
+  }
+`
+
+export const CustomControls = styled.div<CustomControlsProps>`
+  position: absolute;
+  height: 100%;
+  bottom: ${({ isFullScreen }) => (isFullScreen ? sizes(10) : sizes(4))};
+  padding: 0 ${sizes(4)};
+  left: 0;
+  z-index: ${zIndex.overlay};
+  display: flex;
+  align-items: flex-end;
+  width: 100%;
+  opacity: 0;
+  transition: transform 200ms ${transitions.easing}, opacity 200ms ${transitions.easing};
+`
+
+export const ControlButton = styled.button`
+  margin-right: ${sizes(2)};
+  cursor: pointer;
+  border: none;
+  border-radius: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: ${sizes(2)};
+  transition: background-color ${transitions.timings.player} ${transitions.easing} !important;
+
+  & > svg {
+    filter: drop-shadow(0 1px 2px ${colors.transparentBlack[32]});
+  }
+
+  :hover {
+    background-color: ${colors.transparentPrimary[18]};
+    backdrop-filter: blur(${sizes(8)});
+  }
+
+  :active {
+    background-color: ${colors.transparentPrimary[10]};
+  }
+
+  ${focusStyles}
+`
+
+export const VolumeSliderContainer = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+export const thumbStyles = css`
+  border: none;
+  background: ${colors.white};
+  -webkit-appearance: none;
```

Comment:
Shouldn't it be handled by autoprefixer? If not, shouldn't we also use `-moz-appearance`?

Reply:
yeah, you're right I just copied that thoughtlessly from somewhere:D